app/controllers/games_controller.rb

```diff
@@ -1,10 +1,13 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!
+
   def new
     @game = Game.new
   end
 
   def create
     @game = Game.create(game_params)
+    @game.players.create(game_id: @game.id, user_id: current_user.id, color: [:black, :white].sample)
```

Comment:
Yeah I guess so, that does seem to work in rails console if I'm testing it correctly...I'll adjust it before I make this PR ready for review, just waiting on PR #29 since that adds the user sign-in requirement. Thanks Willem!